tests/7.AlgoSDK.js

```diff
@@ -83,6 +83,33 @@ describe('Algosdk (AKA end to end)', function () {
             let tx_golden = "5FJDJD5LMZC3EHUYYJNH5I23U4X6H2KXABNDGPIL557ZMJ33GZHQ";
             assert.deepStrictEqual(js_dec.txID, tx_golden);
         });
+
+        it('should return a blob that matches the go code when adding a lease', function () {
```

Comment:
It would be better the create the transaction using the `MakePaymentTransaction` and `txn.addLease` functions rather than generating the object yourself.